components/script/dom/audiolistener.rs

```diff
@@ -181,4 +186,35 @@ impl AudioListenerMethods for AudioListener {
     fn UpZ(&self) -> DomRoot<AudioParam> {
         DomRoot::from_ref(&self.up_z)
     }
+
+    //https://webaudio.github.io/web-audio-api/#dom-audiolistener-setorientation
+    fn SetOrientation(
+        &self,
+        x: Finite<f32>,
+        y: Finite<f32>,
+        z: Finite<f32>,
+        xUp: Finite<f32>,
+        zUp: Finite<f32>,
+    ) -> Fallible<DomRoot<AudioListener>> {
+        self.forward_x.SetValue(x);
+        self.forward_y.SetValue(y);
+        self.forward_z.SetValue(z);
+        self.up_x.SetValue(xUp);
+        self.up_y.SetValue(yUp);
```

Comment:
yUp doesn't exist?

Reply:
Thanks for pointing out, I have fixed It and addressed the errors from ./mach test-tidy. Can you check now?